.github/workflows/testing.yml

```diff
@@ -49,6 +49,10 @@ jobs:
           git config --global user.name "github-actions[bot]"
           git config --global user.email "github-actions[bot]@users.noreply.github.com"
 
+      - if: runner.os == 'Windows'
+        name: Enable long path support on Windows
+        run: git config --global core.longpaths true
```

Comment:
And voila, it was not lost after all (since was in base whole time)

We will 'git' this addition anyway though we don't currently need it